internal/pkg/template/workload.go

```diff
@@ -143,6 +143,14 @@ type SidecarOpts struct {
 	Command              []string
 	HealthCheck          *ContainerHealthCheck
 	EnvAddonsFeatureFlag bool
+	PortMappings         []*PortMapping
```

Comment:
I wonder if the struct would be more readable if we grouped all the fields for the main container together. Both in the `stack` pkg here but also in `template.WorkloadOpts`.

For example:
```go
type WorkloadOpts struct {
//...
EnvVersion: s.rc.EnvVersion,

// Configuration for the main container.
PortMappings  []PortMapping
Variables         map[string]Variable
Secrets           map[string]Secret
Command      []string
Entrypoint    []string

// Other configurations...
}
```